core/src/main/java/com/datastax/oss/driver/internal/core/os/Native.java

```diff
@@ -84,11 +49,7 @@ public static int getProcessId() {
    * @return {@code true} if JNR {@link Platform} class is loaded.
    */
   public static boolean isPlatformAvailable() {
```

Comment:
Unfortunate naming here, this should be called `isCpuAvailable`.

We could take advantage of this ticket to rename it, it's internal code so it won't break the API.

Reply:
I agree re: unfortunate naming and I'm happy to change it.